src/App.js

```diff
@@ -101,30 +101,59 @@ const App = () => {
             <Text code className="date">
               {date}
             </Text>
-            <a
-              href={`https://twitter.com/intent/tweet?text=I%20joined%20GitHub%20on%20${date}%20🎉%0aFind%20yours%20at:%20https://www.octocatday.com%20😎`}
-              target="popup"
-            >
-              <svg
-                onClick={() =>
-                  window.open(
-                    `https://twitter.com/intent/tweet?text=I%20joined%20GitHub%20on%20${date}%20🎉%0aFind%20yours%20at:%20https://www.octocatday.com%20😎`,
-                    "Share on Twitter",
-                    "width=400,height=600"
-                  )
-                }
-                style={{ marginTop: 20 }}
-                xmlns="http://www.w3.org/2000/svg"
-                width="30"
-                height="30"
-                viewBox="0 0 24 24"
+            <div>
+              <a
+                style={{ marginRight: 10 }}
+                href={`http://www.facebook.com/sharer.php?u=http://octocatday.com`}
+                target="popup"
               >
-                <path
-                  style={{ fill: "#00acee" }}
-                  d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm6.066 9.645c.183 4.04-2.83 8.544-8.164 8.544-1.622 0-3.131-.476-4.402-1.291 1.524.18 3.045-.244 4.252-1.189-1.256-.023-2.317-.854-2.684-1.995.451.086.895.061 1.298-.049-1.381-.278-2.335-1.522-2.304-2.853.388.215.83.344 1.301.359-1.279-.855-1.641-2.544-.889-3.835 1.416 1.738 3.533 2.881 5.92 3.001-.419-1.796.944-3.527 2.799-3.527.825 0 1.572.349 2.096.907.654-.128 1.27-.368 1.824-.697-.215.671-.67 1.233-1.263 1.589.581-.07 1.135-.224 1.649-.453-.384.578-.87 1.084-1.433 1.489z"
-                />
-              </svg>
-            </a>
+                <svg
+                  onClick={() =>
```

Comment:
SVG `onClick` is repetitive. Why do we need that?

Reply:
![Screenshot from 2019-10-05 01-18-32](https://user-images.githubusercontent.com/26611847/66237275-205c7780-e70e-11e9-9bce-5dbf087be725.png)

@ahmadawais  One is for Facebook Icon and the other one is for Twitter

Reply:
I see that. But we don't need repeated links. 

![x](https://on.ahmda.ws/4e0f71/c)

Link inside the anchor `<a href=` is enough. It will span the SVG since that's its child element.

Reply:
Ah okay, that one.

It is actually opening a sharable URL in a pop-up window while staying on the same page.

and if encase popup got blocked by the browser we still have the HTML based link which will open in a new tab. 

I hope that makes sense :)